lib/tasks/loomio.rake

```diff
@@ -1,4 +1,31 @@
 namespace :loomio do
+  task :deploy do
```

Comment:
@gdpelican simple move of deploy to ruby code. working for deploy to loomio-clone